docs/user/trusted-publishers/troubleshooting.md

```diff
@@ -6,6 +6,20 @@ title: Troubleshooting
 
 # Troubleshooting
 
+## Ratelimiting
+
+PyPI currently imposes ratelimits on trusted publisher registration: no more
+than 100 publishers can be registered by a single user or IP address within a 24
+hour window.
+
+This should be more than sufficient for most users (since publisher
+registration should happen rarely relative to publisher use), but maintainers
+with large numbers of projects or who access PyPI via a shared IP address
+may run into ratelimiting errors. If this happens to you, please wait 24 hours,
+try again, and then contact PyPI's admins if the problem persists.
```

Comment:
This should link to somewhere that describes how to contact admins.

Reply:
The FAQs don't contain a dedicated "contact" section but they do link directly to the admin email, so I've done that here as well. Let me know if that works for you!